src/parsers/manifestjson.js

```diff
@@ -392,6 +394,87 @@ export default class ManifestJSONParser extends JSONParser {
     return Promise.all(promises);
   }
 
+  validateStaticThemeImages() {
+    const promises = [];
+    const themeImages = this.parsedJSON.theme && this.parsedJSON.theme.images;
+
+    const validateThemeImage = async (imagePath, manifestPropName) => {
```

Comment:
Could this be put onto `ManifestJSONParser` instead of being created in the call to `validateStaticThemeImages`?

Reply:
sure thing, I've extracted this method in the updated version of this PR.